### DISCUSSION TITLE
Group's roles are not displayed after assigning

### DISCUSSION THREAD

**Discussion by Alexcei88** [keycloak/keycloak#11737] [Archive]
Step to reproduce:
1. Create group
2. Open "Role mapping" tab and assign any role to the group
3. Assigned role is displayed in the group, all ok
4. Then refresh a Groups page
5. Open the modified group again
6. Switch back to Role mapping tab
7. I can see label "no Roles" although the group has the role assigned on 2 step. I expected to see the assigned role.

**Comment by ssilvert** [comment]
@Alexcei88 I tried this on the latest build of our app and it seems to be working fine.

**Comment by Alexcei88** [comment]
Thank you @ssilvert, I will look forward a new release
